### PR TITLE
Content i18n demo

### PR DESCRIPTION
**Pull request by ValJed**
[PRO-2277](https://linear.app/apostrophecms/issue/PRO-2277/as-a-demo-user-i-can-toggle-to-view-content-in-french-or-english)

## Purpose
Config two locales in i18n module, adds a locale selector in global layout if more than two locales.

## How to test
Check that the local selector shows with the two locales, click to switch from one to another.

### CODE REVIEW

File: modules/@apostrophecms/i18n/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  options: {
+    locales: {
+      en: {
+        label: 'English'
+      },
+      fr: {
+        label: 'French',
+        prefix: '/fr'
+      }
+      // 'en-CA': {
```
Comment (ValJed) [inline modules/@apostrophecms/i18n/index.js:11]: Not sure about what do do of this for now? I guess we can delete it.

File: modules/asset/ui/src/_locales.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    window.addEventListener('click', clickOutside);
+
+    function toggleLocales () {
+      const expanded = toggler.getAttribute('aria-expanded') === 'true' || false;
+
+      toggler.setAttribute('aria-expanded', !expanded);
+      localeList.hidden = expanded;
+    }
+
+    function clickOutside ({ target }) {
```
Comment (ValJed) [inline modules/asset/ui/src/_locales.js:22]: Simply close the dropdown if clicking outside.

File: modules/@apostrophecms/i18n/index.js (hunk 1 above)
Comment (boutell) [reply modules/@apostrophecms/i18n/index.js:11]: It's OK to remove the other locales.

**Review by boutell — APPROVED** [review]

File: modules/asset/ui/src/_locales.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+.locales__toggler {
+  cursor: pointer;
+
+  &[aria-expanded="true"] {
+    border-radius: 10px 10px 0px;
+  }
+}
+
+.locales__list {
```
Comment (abea) [inline modules/asset/ui/src/_locales.scss:16]: The bottom-left corner looks odd. Also that the text is not aligned the same way. Let me know if this is from a design.

Screen Shot 2021-11-29 at 12 57 49 PM

File: views/locales.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<div class="locales" data-locales>
+  <button class="locales__toggler button button--small" data-locales-toggle aria-expanded="false">
```
Comment (abea) [inline views/locales.html:2]: This seems like a good place to use `aria-controls` https://accessibilityresources.org/aria-controls

**Review by abea — CHANGES_REQUESTED** [review]

**Pull request merged**